segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java

```diff
@@ -37,6 +37,9 @@
     public static final Property<Integer> ZK_SESSION_TIMEOUT_MS = Property.named("zkSessionTimeoutMs", 10000);
     public static final Property<String> CLUSTER_NAME = Property.named("clusterName", "pravega-cluster");
     public static final Property<String> STORAGE_IMPLEMENTATION = Property.named("storageImplementation", StorageTypes.INMEMORY.toString());
+    public static final Property<Boolean> ENABLE_HEALTHREPORTING = Property.named("enableHealthReporting", false);
+    public static final Property<String> HEALTH_REPORTER_IPADDRESS = Property.named("healthReporterIPAddress", "localhost");
+    public static final Property<Integer> HEALTH_REPORTER_PORT = Property.named("HealthReporterPort", 7070);
```

Comment:
In ZK, we use the same client port to void having to ask the application to open yet another port. This is important, for example, to avoid having to open yet another port in the firewall. Can we avoid having to open another port?